samples/Datadog Sample/ProjectSettings/ProjectSettings.asset

```diff
@@ -847,7 +847,7 @@ PlayerSettings:
   suppressCommonWarnings: 1
   allowUnsafeCode: 0
   useDeterministicCompilation: 1
-  additionalIl2CppArgs: '  --emit-source-mapping'
```

Comment:
is it related to the change?

Reply:
No but removing it is probably a good idea.